ui/static/app/app.js

```diff
@@ -18,4 +18,9 @@ angular.module("sevenWonder", ['ui.router'])
         templateUrl: "app/module/directives/form-game-room/create-game-room.html",
         controller: 'createGameRoomCtrl'
       })
+      .state('join-room', {
+        url: "/join-room",
+        templateUrl: "app/module/directives/join-room/join.html",
```

Comment:
The state template shouldn't be a directive template, you should create a new page where the directive is added. Coordinate with Diego about this.